silo-core/contracts/utils/liquidationHelper/DexSwap.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {IERC20} from "openzeppelin5/token/ERC20/IERC20.sol";
+
+import {ILiquidationHelper} from "../../interfaces/ILiquidationHelper.sol";
+
+import "../../lib/RevertLib.sol";
+
+/// @dev Based on demo contract that swaps its ERC20 balance for another ERC20.
+/// demo source: https://github.com/0xProject/0x-api-starter-guide-code/blob/master/contracts/SimpleTokenSwap.sol
+contract DexSwap {
+    using RevertLib for bytes;
+
+    /// @dev 0x ExchangeProxy address.
+    /// See https://docs.0x.org/developer-resources/contract-addresses
+    /// The `to` field from the API response, but at the same time,
+    /// TODO: maybe unit test that will check, if it does not changed?
+    // solhint-disable-next-line var-name-mixedcase
+    address public immutable EXCHANGE_PROXY;
+
+    event BoughtTokens(address sellToken, address buyToken, uint256 boughtAmount);
+
+    error AddressZero();
+    error TargetNotExchangeProxy();
+    error ApprovalFailed();
```

Comment:
Do we need these errors? I can't find a place where we use them.
```suggestion
```

Reply:
left from copy/pasta, removed